README.md

```diff
@@ -29,35 +27,17 @@ where everything falls into place in the new approach.
 | `{integration}/test_check.py` | `{integration}/tests/*.py`                        |
 | n/a                           | `{integration}/setup.py`                          |
 
-## A note about installing
```

Comment:
I find this confusing, we'll be more precise once the "single check update" feature hits GA